test/elixir_console_web/live/console_live_test.exs

```diff
@@ -28,8 +28,10 @@ defmodule ElixirConsoleWeb.ConsoleLiveTest do
   describe "sending a second valid command" do
     setup(%{conn: conn}) do
       {:ok, view, _html} = live(conn, "/")
-      render_submit([view, "#command_input"], "execute", %{"command" => "a = 3"})
-      render_submit([view, "#command_input"], "execute", %{"command" => "nil"})
+
+      form = element(view, "#command_input")
```

Comment:
Just a note, not scope of this PR, but maybe would me more readable/legible if the `form`'s DOM ID is `#command_form` instead of `#command_input` so isn't confused with the actual form input?

Reply:
Yeah we could make the names less confusing. Now the form is called command_input because it's the root element of the component called CommandInputComponent. So maybe we could call it CommandComponent and use the ids form and input in the html.
Agree that it should be done in another PR